src/database/sqlite.py

```diff
@@ -1220,10 +1182,7 @@ def load_stored_tournaments(self) -> list[StoredTournament]:
             'SELECT * FROM `tournament` ORDER BY `uniq_id`',
             (),
         )
-        stored_tournaments: list[StoredTournament] = [self._row_to_stored_tournament(row) for row in self._fetchall()]
-        for stored_tournament in stored_tournaments:
-            stored_tournament.stored_skipped_rounds = self.load_stored_skipped_rounds(stored_tournament.id)
-        return stored_tournaments
+        return [self._row_to_stored_tournament(row) for row in self._fetchall()]
```

Comment:
Does it need to be a list, or can the consumer transform it into a list?
Assuming appropriate type hints, it would look something like this:
```suggestion
        yield from map(self._row_to_stored_tournament, self._fetchall())
```

Reply:
Methods load_stored_*() are all consumed by load_stored_event() and the results are stored as lists (the goal is to load data at once, store it for future use, and close the the event database as soon as the data is read).
Thus list conversion can be done when storing the data in the StoredEvent or in the load_stored_*(), but it must be done.
Used iterators wherever possible in the last commit.